### PR TITLE
Feature done about and contact page

### PR DESCRIPTION
**Pull request by LAEEQ18**
fixed the header
get in touch open in new-tab
using styled components
move babel-plugin in dev-dependices etc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "gatsby": "^2.24.79",
+    "gatsby-image": "^2.4.21",
+    "gatsby-plugin-sharp": "^2.6.43",
+    "gatsby-plugin-styled-components": "^3.3.14",
+    "gatsby-source-contentful": "^3.0.6",
+    "gatsby-source-filesystem": "^2.3.37",
+    "gatsby-transformer-sharp": "^2.5.20",
+    "global": "^4.4.0",
+    "i": "^0.3.6",
+    "jquery": "^3.5.1",
```
Comment (arif-hanif) [inline package.json:31]: remove

**Comment by arif-hanif** [comment]
@LAEEQ18  remove these files 
image

File: package.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    "react-dom": "^16.14.0",
+    "react-is": "^16.13.1",
+    "styled-components": "^5.2.1",
+    "typescript": "3.7.5"
+  },
+  "devDependencies": {
+    
+   
+    
+
```
Comment (arif-hanif) [inline package.json:44]: remove white spaces

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+const Footers = () => (
+  <Menu
+    theme="light"
+    mode="horizontal"
+    defaultSelectedKeys={['1']}
+    style={{
+      width: '100%',
+      background: 'grey',
+
```
Comment (arif-hanif) [inline src/components/Footer.js:17]: white space

File: src/components/Footer.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+          width="110"
+          height="82"
+          style={{
+            display: 'block',
+            marginLeft: 'auto',
+            marginRight: 'auto',
+            marginTop: '20px',
+          }}
+        />
+
```
Comment (arif-hanif) [inline src/components/Footer.js:37]: white space

File: src/components/layout.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        height: '11%',
+        backgroundColor: 'white',
+      }}
+    >
+      <Navbar />
+    </Header>
+    <Content style={{ backgroundColor: 'white', paddingBottom: '30px' }}>
+      <main> {children} </main>
+    </Content>
+
```
Comment (arif-hanif) [inline src/components/layout.js:26]: White space

File: src/components/layout.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    <Footers />
```
Comment (arif-hanif) [inline src/components/layout.js:27]: rename this Footer

File: src/components/layout.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Layout } from 'antd'
+import Navbar from './Navbar'
+import Footers from './Footer'
+import { LayoutCointainer } from './layout.style'
+import 'antd/dist/antd.css'
+
+const { Header, Content } = Layout
+
+const layout = ({ children }) => (
```
Comment (arif-hanif) [inline src/components/layout.js:10]: const Layout

File: src/components/Footer.js [hunk 8]
```diff
@@ -0,0 +1,169 @@
+import { Link } from 'gatsby'
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Row, Col, Menu } from 'antd'
+import '../utils/font-awesome'
+import styled from 'styled-components'
+import intel from '../images/intelf.png'
+
+const Footers = () => (
```
Comment (arif-hanif) [inline src/components/Footer.js:9]: const Footer

File: src/pages/index.js [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+            fontWeight: '600',
+            margin: '30px',
+          }}
+        >
+          {' '}
+          Privacy Policy{' '}
+        </h1>
+
+        <h2> Privacy Notice </h2>
+        <p>
```
Comment (arif-hanif) [inline src/pages/index.js:30]: Move privacy policy to seperate component and link from footer, just put place holder of home.

**Review by arif-hanif — CHANGES_REQUESTED** [review]

**Comment by hanif-arif** [comment]
@LAEEQ18 navbar is way to large when you scroll down, it should only be about 60-80px

**Comment by hanif-arif** [comment]
When user clicks on Navbar, the navbar should highlight the selected section

File: src/components/Navbar.js [hunk 10]
```diff
@@ -0,0 +1,48 @@
+import { Link } from 'gatsby'
+import React from 'react'
+import { Menu } from 'antd'
+import logo from '../assets/logo.svg'
+
+const Navbar = () => (
```
Comment (hanif-arif) [inline src/components/Navbar.js:6]: Track state of selected menu and highlight the selected menu

File: src/pages/contacts.styled.jsx [hunk 11]
```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components'
+
+export const ButonCon = styled.button`
```
Comment (hanif-arif) [inline src/pages/contacts.styled.jsx:3]: StyledButton

**Review by hanif-arif — CHANGES_REQUESTED** [review]

**Comment by hanif-arif** [comment]
@LAEEQ18 you also have alot of warnings in the console that need to be cleared
image

**Comment by LAEEQ18** [comment]
> When user clicks on Navbar, the navbar should highlight the selected section

yes but for one click dont'know why this were not enabled on 2 click  it get sleceted i also give him active class

File: src/components/layout.js (hunk 7 above)
Comment (LAEEQ18) [reply src/components/layout.js:10]: if using Layout then it will give error on **Children** prop Error

Comment (LAEEQ18) [reply src/components/layout.js:10]: Layout = ({ children })
**'children' is missing in props validation**
so i use
```suggestion
const layout = ({ children }) => (

```

File: src/components/Navbar.js (hunk 10 above)
Comment (LAEEQ18) [reply src/components/Navbar.js:14]: the selected state is now highlited

**Review by hanif-arif — APPROVED** [review]

**Pull request merged**